listener.go

```diff
@@ -34,7 +34,7 @@ func (h *apigeeSyncHandler) initListener(services apid.Services) {
 }
 
 func (h *apigeeSyncHandler) String() string {
-	return "verifyAPIKey"
+	return "apiMetadata"
```

Comment:
Sounds like this string literal also repeated, what is best practice in `Golang` world for such usage?

Reply:
This `String()` implements an interface for apid-core event handler.
It provides logging info for apid-core to log system events life-cycle.
By convention it's plugin name without "apid" prefix.